python/hangover.py

```diff
@@ -13,18 +13,68 @@
 
 # returns 'None' if the key doesn't exist
 TWILIO_ACCOUNT_SID = os.environ.get('TWILIO_ACCOUNT_SID')
-TWILIO_AUTH_TOKEN  = os.environ.get('TWILIO_AUTH_TOKEN')
+TWILIO_AUTH_TOKEN = os.environ.get('TWILIO_AUTH_TOKEN')
 
 # Phone numbers
-my_number      = '+xxx'
+my_number = '+xxx'
 number_of_boss = '+xxx'
 
+# Shout out to https://idontlike.work
 excuses = [
-  'Locked out',
-  'Pipes broke',
-  'Food poisoning',
-  'Not feeling well'
-]
+    "Locked out",
+    "Pipes broke",
+    "Food poisoning",
+    "Not feeling well",
+    "My gas cylinder is leaking. Need to get it fixed.",
+    "Got some pain in the back when I woke up in the morning. Need to visit a doctor.",
+    "My cat somehow ate marijuana and is behaving wierd.",
```

Comment:
```suggestion
    "My cat somehow ate marijuana and is behaving weird.",
```

Reply:
@viccomx that could be the cat writing it 🤣